explainers/subresource-loading.md

```diff
@@ -156,6 +156,10 @@ subresources, `a.js`, `b.js`, and `c.png`, from the bundle.
 `urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6` is also loaded from
 the bundle, and a subframe is instantiated as an
 [opaque origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque) frame.
+Even if a custom scheme handler for `urn` schme was registered by
```

Comment:
How about not mentioning "urn" scheme here? I think it is redundant.
Can we simply say, for example, "The browser must ignore any custom registered protocol handler in loading a resource from a bundle"?

Reply:
I'm not sure whether it is a good idea or not to ignore a custom registered protocol handler in loading a subresource from a bundle. That needs a discussion, however, it seems much *safer* to ignore any custom registered protocol handler until we have a consensus.

Reply:
Done